### ISSUE TITLE
there is something wrong on lg g3

### ISSUE THREAD

**Issue by Apcan** [roughike/BottomBar#151]
when i use it on my project there ,![](http://7xrn7f.com1.z0.glb.clouddn.com/16-4-11/71850377.jpg)

**Comment by roughike** [comment]
Could you provide some sample Java and XML code?

**Comment by behelit** [comment]
I'm having the same issue, but it is only appearing on my Android N device, API 17 seems OK.

**MainActivity**

```
@Override
protected void onCreate(Bundle savedInstanceState)
{
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);

     mBottomBar = BottomBar.attach(this, savedInstanceState);
```


and I use this to set the fragment
```
getSupportFragmentManager().beginTransaction()
      .replace(R.id.fragment_container, ConnectFragment.newInstance()).commit();
```


**R.layout.activity_main**
```
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <FrameLayout
        android:id="@+id/fragment_container"
        android:layout_width="match_parent"
        android:layout_height="match_parent"/>
</RelativeLayout>
```

**R.layout.fragment_connect**
```
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                xmlns:app="http://schemas.android.com/apk/res-auto">

    <android.support.v7.widget.Toolbar
        android:id="@+id/connectToolbar"
        app:title="@string/bottomnav_control"
        android:layout_width="match_parent"
        android:layout_height="?attr/actionBarSize"
        android:background="@color/bottomnavConnect"
        android:layout_alignParentTop="true"
        app:elevation="4dp"
        />

    <android.support.v7.widget.RecyclerView
        android:id="@+id/connectRecyclerView"
        android:layout_below="@+id/connectToolbar"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:background="@color/cardview_dark_background"/>

</RelativeLayout>
```

I was previously using CoordinatorLayout's but changed to RelativeLayout's to see if it would resolve the problem.
I've also tried attaching the bottomBar directly to the fragment_container, as suggested in some other issues, but it produces the same result.

**Comment by Apcan** [comment]
`<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    tools:context=".activity.MainActivity">

    <FrameLayout
        android:id="@+id/fm_content"
        android:layout_width="match_parent"
        android:layout_height="match_parent" />
</RelativeLayout>
`
` @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.layout_activity_main);
        ButterKnife.bind(this);
        initLocation();
        initFm();
        initData();
        mBottomBar = BottomBar.attach(this, savedInstanceState);`

there is the layout and activity of mine

then i found that the wrong layout is bb_bottom_bar_item_container

**Comment by roughike** [comment]
Should be fixed in the newest versions (2.0 and up), since adding the BottomBar to your layout isn't so magical anymore.
